docs/spec_draft.md

```diff
@@ -22,7 +22,7 @@ Following are the supported element types in StableHLO:
     identical behavior for underflows, overflows, and NaNs. However, `bf16`
     handles denormals differently from `f32`: it flushes them to zero.
   * **Complex types** represents a pair of floating-point types. Supported ones
- are `c64` (represents paired `f32`) and `c128` (represents paired `f64`).
+    are `c64` (represents paired `f32`) and `c128` (represents paired `f64`).
```

Comment:
Do we need this space?

Reply:
This is consistent with how other bullet lists are formatted in the spec, with lines starting where the first line (right after `* `) starts.